cmd2/cmd2.py

```diff
@@ -1680,9 +1680,7 @@ def onecmd_plus_hooks(self, line: str, *, add_to_history: bool = True,
         except Exception as ex:
             self.pexcept(ex)
         finally:
-            stop = self._run_cmdfinalization_hooks(stop, statement)
-
-        return stop
+            return self._run_cmdfinalization_hooks(stop, statement)
```

Comment:
It looks like this change is causing two unit tests to fail:
1. `test_runcmds_plus_hooks_ctrl_c`
2. `test_generate_transcript_stop`